homeassistant/components/jvc_projector/strings.json

```diff
@@ -37,6 +37,18 @@
       "jvc_power": {
         "name": "[%key:component::sensor::entity_component::power::name%]"
       }
+    },
+    "sensor": {
+      "jvc_power_status": {
+        "name": "Power status",
+        "state": {
+          "standby": "StandBy",
+          "on" : "On",
+          "warming": "Warming",
+          "cooling" : "Cooling",
```

Comment:
Extra space in front of colon.

Reply:
Fixed

Reply:
Thanks! Looking good. Hopefully this PR is gtg!